src/components/modals/Modal.js

```diff
@@ -1,11 +1,14 @@
 import PropTypes from 'prop-types';
-import React, {useEffect} from 'react';
+import React, {createContext, useContext, useEffect} from 'react';
```

Comment:
it's to make sure the modal is rendered in the right div for storybook. so that we can make interaction tests with `within(canvasElement)` because the div for the modal by defaults gets added to the `<body>` element, which is the container for `canvasElement`.

Just things that Storybook makes a bit harder than you'd like :grimacing: